MediaBrowser.Providers/Plugins/Tmdb/Api/TmdbController.cs

```diff
@@ -0,0 +1,41 @@
+using System.Net.Mime;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using TMDbLib.Objects.General;
+
+namespace MediaBrowser.Providers.Plugins.Tmdb.Api
+{
+    /// <summary>
+    /// The TMDb api controller.
+    /// </summary>
+    [ApiController]
+    [Authorize(Policy = "DefaultAuthorization")]
+    [Route("[controller]")]
```

Comment:
Typically the route is `<PluginNamespace>`, but we haven't standardized it (we really should though).

Reply:
`[controller]` resolves to "tmdb", you're saying I should set it to `[Route("tmdb")]` directly?

Checking a few plugins that have controllers:
[Trakt](https://github.com/jellyfin/jellyfin-plugin-trakt/blob/a581d4dce1bbc3a3632841f2b6fb219f04ce4329/Trakt/Api/TraktController.cs#L23) and [TmdbBoxSet](https://github.com/jellyfin/jellyfin-plugin-tmdbboxsets/blob/9ede4a11f6a525c360bd21628a9dad689a9f086c/Jellyfin.Plugin.TMDbBoxSets/Api/TMDbBoxSetsController.cs#L17) set `Route("[controller]")` like I did here.

[OpenSubtitles](https://github.com/jellyfin/jellyfin-plugin-opensubtitles/blob/1e83683bc3d02a25422d9b9302165164583deac5/Jellyfin.Plugin.OpenSubtitles/API/OpenSubtitlesController.cs#L19) and [KodiSyncQueue](https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue/blob/7af5fef9eaac55200e67786646a077350340a36e/Jellyfin.Plugin.KodiSyncQueue/API/KodiSyncQueueController.cs#L23) don't set a `Route` at all.

[AutoOrganize](https://github.com/jellyfin/jellyfin-plugin-autoorganize/blob/73cc10464fd514d73f0eae8f1b80c2f30dc16b35/Emby.AutoOrganize/Api/FileOrganizationController.cs#L19) sets it to `Library/FileOrganizations`